Modules/immutablemodule.c

```diff
@@ -0,0 +1,197 @@
+/* immutable module */
+
+#ifndef Py_BUILD_CORE_BUILTIN
+#  define Py_BUILD_CORE_MODULE 1
+#endif
+
+#define MODULE_VERSION "1.0"
+
+#include "Python.h"
+#include <stdbool.h>
+#include "pycore_object.h"
+
+/*[clinic input]
+module immutable
+[clinic start generated code]*/
+/*[clinic end generated code: output=da39a3ee5e6b4b0d input=46b92e14e140418a]*/
+
+#include "clinic/immutablemodule.c.h"
+
+typedef struct {
+    PyObject *not_freezable_error_obj;
```

Comment:
Is this object still used? Looking at the code, it seems to only be initialized and cleared, but never used outsite of this